src/lib/rest/RestService.cpp

```diff
@@ -443,7 +444,7 @@ static bool compErrorDetect
 * we need to know if it's v1 or v2.
 *
 * This is very easy, we just look at the first component of the URI PATH, and
-* if it is "v2" or "V2", then case-sensitive checks are used.
+* if it is 2 or 2, then case-sensitive checks are used.
```

Comment:
The whole comment preamble can be removed. API detection has been unified in just one place: `apiVersionGet`. In that function is where this should be explained (actually, it is, which simpler comments).

In fact, note that before this PR, the restService() function was using at the same time the ciP->apiVersion in some places and the apiVersion caculated looking to comp[] in another place. Crazy... :)